### PR TITLE
added login

### PR DESCRIPTION
**Pull request by sushmita12345**
Added login page to the E-commerce website.

Sample
Screenshot 2022-02-11 at 2 10 12 AM

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
looks good to me

File: assets/css/login.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+.container-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--white);
+  padding: 3rem 5rem;
+  border-radius: 0.5rem;
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 1px 4px;
+}
+
```
Comment (PuneetP16) [inline assets/css/login.css]: Could use variables for colours or mention some comment on the same if just used once.

**Review by PuneetP16 — APPROVED** [review]
GTM

File: assets/css/login.css (hunk 1 above)
Comment (sushmita12345) [reply assets/css/login.css]: Ok, I'll update.

**Pull request merged**